canonical-web-and-design/maas-ui
# Intro name form

## Done

- Add the MAAS name form to the intro.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Use the CLI to reset the intro: https://github.com/canonical-web-and-design/maas-ui/pull/2854/files#diff-9769a3f5504af3cd735e08f078b94f827bdd51eacac2e9d80ac9761bc485171fR552
- Visit /r/intro
- Enter a name and submit.
- Your window title should show your new MAAS name.
- Refresh the page, it shouldn't try and display the intro again.

## Fixes

Fixes: https://github.com/canonical-web-and-design/app-squad/issues/89.
